simulation/behavior_tree_plugin/config/vehicle_entity_behavior.xml

```diff
@@ -15,6 +15,7 @@
             entity_type_list="{entity_type_list}"
             lane_change_parameters="{lane_change_parameters}"
             route_lanelets="{route_lanelets}"
+            common_spline="{common_spline}"
```

Comment:
What is the meaning of "common"?

Reply:
I think this naming is too ambiguous.

Reply:
By "common" I meant "shared by all the actions". This spline is precalculated and passed to all the actions to avoid redundant calculations.

Reply:
What name would be more appropriate in your opinion?

Reply:
How about "reference_trajectory" ?

Reply:
Seems better than "common_spline", I will change it